roles/servicetelemetry/defaults/main.yml

```diff
@@ -56,6 +54,9 @@ servicetelemetry_defaults:
           persistent:
             storage_class: ""
             pvc_storage_request: 20Gi
+        certificates:
+          endpoint_cert_duration: 70080h
+          ca_cert_duration: 70080h
```

Comment:
You were using the defaults here instead of overriding them because of the casing issue earlier (untested).

Reply:
Gotcha, fixing it